src/stories/Library/material-header/material-header.scss

```diff
@@ -21,10 +21,8 @@
       padding: 0;
     }
 
-    & .cover-container {
-      & > span.cover {
-        box-shadow: 0 4px 40px rgba(0, 0, 0, 0.2);
-      }
+    .cover__img {
+      box-shadow: 0 4px 40px rgba(0, 0, 0, 0.2);
```

Comment:
Can we remove this and add the `cover__img--shadow` instead?

There might be a slight change in the definition (0.2 vs 15%) but I wonder if difference is actually intentional.